src/game/moveFinding/findKnightBaseMoves.js

```diff
@@ -0,0 +1,29 @@
+import BoardCoordinates from "../board/boardCoordinates";
+class Knight {
+    
+    outOfBounds(i, j) {
+        return ( i < 0 || i >= 8 || j < 0 || j >= 8 );
+    }
+
+    findMovesx(boardSituation, startingCoordinates, pieceColor){
+        const knightMoves = [
+            [-1, -2], [-2, -1], [1, -2], [-2, 1],
+            [2, -1], [-1, 2], [2, 1], [1, 2]
+        ];
+        var moveX = startingCoordinates.x;

```

Comment:
Lepiej jest unikać deklarowania zmiennych jako **var**, jeśli nie jest to potrzebne - w tym wypadku zas zdecydowanie nie jest. Lepiej jest wyrobić sobie odruch stosowania const (gdy to tylko możliwe) i let (gdy możliwe nie jest), var pozostawiając na wyjątkowe okazje gdy chce się skorzystać ze specyficznych właściwości tego rodzaju zmiennej.